scaling_performance/optimizing_storage.adoc

```diff
@@ -11,25 +11,185 @@
 
 toc::[]
 
-== Optimizing Storage
+This topic provides information for optimizing {product-title} storage.
+
+[[general-storage-guidelines]]
+== General Storage Guidelines
+
+The following table lists the persistent storage technology that {product-title} can be used with.
+
+.Available storage options
+[cols="1,4,3",options="header"]
+|===
+| Storage type | Description | Examples
+
+| Block
+| This type of storage is presented to the operating system as a block device. It is
```

Comment:
Is this necessary? "This type of storage is presented to the operating system as a block device."

Again, try to minimize the words. A table is supposed to be easy to browse through:

"Referred to as a Storage Area Network (SAN). This type of storage is suitable for applications that need full control of storage and operate at a low level on files bypassing the file system.  It is usually a non-shareable type of storage, which means that only one client at a time can mount an endpoint of this type."

Consider doing the same will the other descriptions below.

Reply:
I was intentionally this verbose because not all OpenShift administrators will have strong Storage experience.  The verbosity here provides baseline understanding w/o resorting to citation heavy descriptions.

Reply:
I think we can still get across the same amount of information without so many extraneous words.